x-pack/legacy/plugins/siem/public/pages/case/components/all_cases/columns.test.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
+
+import { ServiceNowColumn } from './columns';
+
+import { useGetCasesMockState } from '../__mock__/case_data';
+
+describe('ServiceNowColumn ', () => {
```

Comment:
What do you think about adding tests that check the logic at the rest of the columns? For example, a test that `getEmptyTagValue` is rendered when the tags are null or a test that checks the open/close states.

Reply:
I think that we need to test `renderStringField` function

Reply:
these are done in `all_cases/index.test.tsx`
`it('should render empty fields'`